### PR TITLE
Add manual-email-sender with batchSend

### PR DESCRIPTION
**Pull request by ocodista**

### CODE REVIEW

File: apps/manual-email-sender/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "email-sender",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "add-unconfirmed-to-queue": "ts-node src/confirmation-email/addUnconfirmedToQueue.ts",
+    "send-unconfirmed-from-queue": "ts-node src/confirmation-email/sendConfirmationEmailsFromQueue.ts"
```
Comment (ocodista) [inline apps/manual-email-sender/package.json:8]:
```suggestion
    "send-openings": "ts-node src/openings-email/sendEmail.ts",
```

**Review by anti-duhring — CHANGES_REQUESTED** [review]

File: packages/shared/src/email/openings-email/28-06-2023/roles.ts
[the remark this answers is not in the document]
Comment (ocodista) [reply packages/shared/src/email/openings-email/28-06-2023/roles.ts:1]: > We need this roles file?

It was used for sending the first batch of roles, we can keep until we implement the fetch-roles-from-db feature.

what do you think?

[the remark this answers is not in the document]
Comment (anti-duhring) [reply packages/shared/src/email/openings-email/28-06-2023/roles.ts:1]: No problem at all, we can keep this file

**Comment by anti-duhring** [comment]
@ocodista can you look at what made the deployment fail?

**Comment by ocodista** [comment]
> @ocodista can you look at what made the deployment fail?

It was an error on package.json.

Fixed now, let's see how the new deploy goes

**Pull request merged**